x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/index_lifecycle_summary.js

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Component, Fragment } from 'react';
+import moment from 'moment-timezone';
+import {
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiSpacer,
+  EuiTitle,
+  EuiLink,
+  EuiPopover,
+  EuiPopoverTitle,
+} from '@elastic/eui';
+import { i18n }  from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { getPolicyPath } from '../../services/navigation';
+const HEADERS = {
```

Comment:
Optional: consider wrapping this in a function, to avoid translating on static import.
Based on my conversation [here](https://github.com/elastic/kibana/pull/26416#discussion_r239435127), it's not required for client-side code, but recommended.

Reply:
done